vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -108,6 +110,11 @@ const (
 func main() {
 	klog.InitFlags(nil)
 	kube_flag.InitFlags()
+
+	if len(*vpaObjectNamespace) > 0 && len(*ignoredVpaObjectNamespaces) > 0 {
```

Comment:
Same here, I think the version log should come first.

Reply:
Fixed in 590b4fc00